inductiva/resources/resources.py

```diff
@@ -0,0 +1,43 @@
+"""Methods related to managing executers and resource pools."""
+from typing import Optional
+from uuid import UUID
+
+import inductiva
+from inductiva import api
+from inductiva.resources import Machines
+from inductiva.client import ApiClient, ApiException
+from inductiva.client.apis.tags.executers_api import ExecutersApi
+
+
+def create_resource_pool() -> UUID:
```

Comment:
if this creates an ID change the name of the method to make that clear, otherwise it seems it creates a resource pool (also, see from my other comments, that maybe we should no longer use the term "resource pool" if we are starting to call it "machine group"

Reply:
Yes, I agree, it should have a more explicit name. I will rename it.